### DISCUSSION TITLE
Is there a way to reduce the number of notification emails?

### DISCUSSION THREAD

**Discussion by rado84-github** [community/community#83243] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Every time someone comments my bug report I get an e-mail for each comment. Which can quickly turn into a spam (in a way), if there are 10 or more comments. I checked the settings for notifications but I couldn't find any option that would be even close to "send e-mail once a day per topic/bug commented".
Having one separate e-mail per comment is pointless cuz I'll see all comments once I open the topic or bug report I've opened. Thats why I wish to reduce the notificiation e-mails to once a day per each topic/bug report, if it's possible.

**Comment by the1Riddle** [comment]
Hi,
If you want to manage GitHub discussion and pull request notifications being sent to your email, what I do is to typically navigate to the `notification section` and then `unsubscribe` from specific threads or updates. This ensures a streamlined inbox experience while staying informed about relevant project activities and you will not receive those emails again.

![image](https://github.com/community/community/assets/125451537/2792f269-6e20-4331-b7a6-5ce148874752)

Hope that helps.

**Comment by rado84-github** [comment]
I suppose there isn't a way to be automatically unsubscribed from all past, present and future topics?

**Reply by the1Riddle** [reply]
yeah, sure, haven't found any to do that.

**Reply by the1Riddle** [reply]
maybe if you give it a search on github docs

**Comment by avidenic** [comment]
Getting a daily digest would be really great, instead of getting an email for everything all the time.

**Reply by the1Riddle** [reply]
there is probably a way

**Reply by the1Riddle** [reply]
you can turn off Automatically watch teams, on notifications settings, then on subscriptions, select all the past ones and unsubscribe to them

**Reply by avidenic** [reply]
I'd still like to see them, but not in separate emails.
